examples/games/crystal_ball/lib/crystal_ball.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:ui';
+
+import 'package:crystal_ball/src/game/game.dart';
+
+import 'package:flame/game.dart';
+import 'package:flutter/material.dart';
+
+class CrystalBallWidget extends StatefulWidget {
+  const CrystalBallWidget({super.key});
+
+  static const String description = '''
+    A demonstration of how to leverage the power of Fragment Shaders in Flame.
+    The game is a simple crystal ball that jumps around in a swampy world.
+    It uses custom post process shaders to create fog, fireflies, water reflections
+    and the glowing effect of the crystal ball.
```

Comment:
```suggestion
A demonstration of how to leverage the power of Fragment Shaders in Flame.
The game is a simple crystal ball that jumps around in a swampy world.
It uses custom post process shaders to create fog, fireflies, water reflections
and the glowing effect of the crystal ball.
```
When using triple ticks the space is included too